addon/templates/components/cfb-form-editor/question/default.hbs

```diff
@@ -0,0 +1,11 @@
+{{component labelComponent}}
+
+{{component (get-widget this.field.question)
+  field=this.field
+  disabled=false
+  context=(hash)
+  onSave=this.onUpdate
+}}
+
+{{component hintComponent}}
+{{component errorComponent}}
```

Comment:
I think you could just render the whole `cf-field` component and use the new default component just as wrapper. This way you wouldn't have to assign the errors manually

Reply:
Unfortunately, that doesn't work as expected.

![Screenshot_20210105_103301](https://user-images.githubusercontent.com/65539425/103630131-8ce1aa00-4f41-11eb-922c-f9726f69b96c.png)
(`cf-field` is the second)

Reply:
Ah, now I see the use case.. Fair enough!